extensions/notification/queueProcessor/task.js

```diff
@@ -17,8 +17,26 @@ werelogs.configure({
 try {
     const destination = process.argv[2];
     assert(destination, 'task must be started with a destination as argument');
+    // get destination auth config from environment variables
+    let destinationAuth = {
+        type: process.env.TYPE,
+        ssl: process.env.SSL && process.env.SSL === 'true',
```

Comment:
```suggestion
        ssl: process.env.SSL === 'true',
```
First check can be skipped as undefined does not equal true